cms/djangoapps/contentstore/helpers.py

```diff
@@ -299,13 +302,21 @@ def import_staged_content_from_user_clipboard(parent_key: UsageKey, request) ->
             tags=user_clipboard.content.tags,
         )
 
+        usage_key = new_xblock.scope_ids.usage_id
+        if usage_key.block_type == 'video':
```

Comment:
This overlaps a bit with the changes at https://github.com/openedx/edx-platform/pull/36173. Since there are some refactors to these functions here. I'll wait for the merge of this PR to update https://github.com/openedx/edx-platform/pull/36173 with your changes